pkg/apiserver/diagnose/assets_dev.go

```diff
@@ -0,0 +1,31 @@
+//+build dev
+
+package diagnose
+
+import (
+	"log"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func getDiagnosisAssetsPath() string {
+	path, err := os.Getwd()
+	if err != nil {
+		log.Println(err)
+	}
+	var assetsPath string
+	if strings.HasSuffix(path, "tidb-dashboard") {
+		// dev mode
+		assetsPath = path + "/ui/packages/diagnosis_report/build"
+	} else if strings.HasSuffix(path, "diagnose") {
+		// work with `go generate ./pkg/apiserver/diagnose` in the project root folder
+		assetsPath = path + "/../../../ui/packages/diagnosis_report/build"
```

Comment:
There will be no issues if you use webpack-dev-server to serve this page

Reply:
😂in fact I tried this,  the biggest problem is the homepage value in the package.json won't add to the index.html by webpack-dev-server, it brings more changes for other files. I tried kinds of solutions, and it seems this solution changes code least (maybe I am too weak...)